pyfar/signal.py

```diff
@@ -800,135 +846,141 @@ def __next__(self):
 
 
 def add(data: tuple, domain='freq'):
-    """Add signals and/or array likes.
+    """Add pyfar audio objects, array likes, an scalars.
+
+    Pyfar audio objects are: Signal, TimeData, and FrequencyData.
 
     Parameters
     ----------
     data : tuple of the form (data_1, data_2, ..., data_N)
-        Data to be added. Can contain Signal objects, numbers, and, array
-        likes.
-    domain : 'time', 'freq'
+        Data to be added. Can contain pyfar audio objects, array likes, and
+        scalars. Pyfar audio objects can not be mixed, e.g., TimeData and
+        FrequencyData objects do not work together.
+    domain : 'time', 'freq', optional
         Flag to indicate if the operation should be performed in the time or
         frequency domain. If working in the frequency domain, the FFT
         normalization is removed before the operation (See
```

Comment:
removed = set to 'none'? Is this information important for the user? same holds for other arithmetic operations

Reply:
No, I also it is better this way